src/opera/scripts/pge_main.py

```diff
@@ -177,7 +177,9 @@ def pge_main():
     run_config_filename = os.path.abspath(args.file)
 
     if not os.path.exists(run_config_filename):
-        raise FileNotFoundError(f"Could not find config file: {run_config_filename}")
+        msg = f"Could not find config file: {run_config_filename}: argument entered {run_config_filename}"
```

Comment:
Not sure what's going on here, more leftover debug code? Can these changes be reverted?

Reply:
Done